--- gsmartcontrol-0.8.5.orig/src/applib/storage_detector.cpp	2009-09-05 12:52:01.000000000 +0400
+++ gsmartcontrol-0.8.5/src/applib/storage_detector.cpp	2010-12-09 23:02:54.000000000 +0400
@@ -225,6 +225,7 @@
 		blacklist.push_back("/part[0-9]+$/");  // devfs had them
 		blacklist.push_back("/p[0-9]+$/");  // partitions are usually marked this way
 		blacklist.push_back("/md[0-9]*$/");  // linux software raid
+		blacklist.push_back("/dm-[0-9]*$/");  // linux device mapper
 
 
 		for (unsigned int i = 0; i < lines.size(); ++i) {
